Dockerfile

```diff
@@ -1,55 +1,10 @@
-FROM google/dart:2.5 as dart2
-FROM drydock-prod.workiva.net/workiva/smithy-runner-generator:350667 as build
-
-ARG NPM_TOKEN
-ARG NPM_CONFIG__AUTH
-ARG NPM_CONFIG_REGISTRY=https://npm.workiva.net
-ARG NPM_CONFIG_ALWAYS_AUTH=true
-
-# Build Environment Vars
-ARG BUILD_ID
-ARG BUILD_NUMBER
-ARG BUILD_URL
-ARG GIT_COMMIT
-ARG GIT_BRANCH
-ARG GIT_TAG
-ARG GIT_COMMIT_RANGE
-ARG GIT_HEAD_URL
-ARG GIT_MERGE_HEAD
-ARG GIT_MERGE_BRANCH
-ARG GIT_SSH_KEY
-ARG KNOWN_HOSTS_CONTENT
+FROM drydock-prod.workiva.net/workiva/dart2_base_image:1
 WORKDIR /build/
 ADD . /build/
 
-RUN mkdir /root/.ssh && \
-    echo "$KNOWN_HOSTS_CONTENT" > "/root/.ssh/known_hosts" && \
-    chmod 700 /root/.ssh/ && \
-    umask 0077 && echo "$GIT_SSH_KEY" >/root/.ssh/id_rsa && \
-    eval "$(ssh-agent -s)" && ssh-add /root/.ssh/id_rsa
-RUN echo "installing npm packages"
-RUN npm install
-
-RUN echo "Starting the before_script section" && \
-		dart --version
-
-# Use pub from Dart 2 to initially resolve dependencies since it is much more efficient.
-COPY --from=dart2 /usr/lib/dart /usr/lib/dart2
-RUN echo "Running Dart 2 pub get.." && \
-	_PUB_TEST_SDK_VERSION=1.24.3 timeout 5m /usr/lib/dart2/bin/pub get --no-precompile
-
-RUN pub get && \
-		echo "before_script section completed"
-RUN echo "Starting the script section" && \
-		./tool/check_version.sh && \
-		echo "script section completed"
-ARG BUILD_ARTIFACTS_DART-DEPENDENCIES=/build/pubspec.lock
+RUN pub get
 
 RUN tar -cvzf /build/assets.tar.gz -C lib sockjs.js sockjs_prod.js
 ARG BUILD_ARTIFACTS_CDN=/build/assets.tar.gz
```

Comment:
This appears to be a new artifact. What's the story with it?

Reply:
It was there before: https://github.com/Workiva/sockjs_client_wrapper/blob/master/Dockerfile#L48-L49

This allows us to pull the sockjs script from its own CDN artifact so that it is cached (since it changes so infrequently) compared to pulling it from the wdesk CDN artifact.

Reply:
Oh, duh, I misread the diff. I see now that it isn't new. 🤕 I'm still working on my second cup of coffee this morning.